server/routes/tasks.js

```diff
@@ -0,0 +1,14 @@
+import express from "express";
+import TasksController from "../controllers/tasks.js";
+
+const router = express.Router();
+
+router.get("/", TasksController.getTasks); 
+router.get("/formated", TasksController.getTasksFormated)
```

Comment:
The "/formatted" (sp) might not be necessary. I think that formatting in the front end with util functions is fine so we can keep the backend API clean.